src/test/java/base/BaseTest.java

```diff
@@ -1,71 +1,71 @@
 package base;
 
-import static java.util.Optional.ofNullable;
-import static utils.PropertiesHelper.getInstance;
-
 import manager.PageFactoryManager;
 import org.apache.log4j.PropertyConfigurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.openqa.selenium.WebDriver;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Listeners;
-import org.testng.annotations.Optional;
-import org.testng.annotations.Parameters;
 import pages.HomePage;
 import utils.PropertiesHelper;
-import reporting.TestListener;
 import webDriverFactory.LocalWebDriverCreator;
 import webDriverFactory.RemoteWebDriverCreator;
 import webDriverFactory.WebDriverCreator;
 
-@Listeners({TestListener.class})
+import static java.util.Optional.ofNullable;
+import static utils.PropertiesHelper.getInstance;
+
 public class BaseTest {
-  public static WebDriver driver;
-  public static PropertiesHelper propertiesHelper;
-  protected static HomePage homePage;
-  protected static PageFactoryManager pageFactoryManager;
-  protected static long timeout;
-  protected static String baseUrl;
-  private static String hubUrl;
-  protected final int FIRST = 1;
+    public static WebDriver driver;
+    public static PropertiesHelper propertiesHelper;
+    protected static HomePage homePage;
+    protected static PageFactoryManager pageFactoryManager;
+    protected static String browser;
+    protected static boolean localRun;
+    protected static long timeout;
+    protected static String baseUrl;
+    private static String hubUrl;
+    protected final int FIRST = 1;
 
-  private static void configureLog4j() {
-    PropertyConfigurator.configure(propertiesHelper.getProperty("log4jPropertiesPath"));
-  }
+    private static void configureLog4j() {
+        PropertyConfigurator.configure(propertiesHelper.getProperty("log4jPropertiesPath"));
+    }
 
-  @Parameters({"browser", "localRun"})
-  @BeforeMethod
-  public void setUp(String browser, @Optional("true") boolean localRun) {
-    setUpDriver(browser, localRun);
-    driver.manage().window().maximize();
-    pageFactoryManager = new PageFactoryManager(driver);
-    homePage = pageFactoryManager.getPage(HomePage.class);
-    propertiesHelper = getInstance();
-    configureLog4j();
-    configureConstant();
-  }
+    @BeforeEach
+    public void setUp() {
+        propertiesHelper = getInstance();
+        configureConstant();
+        setUpDriver(browser, localRun);
+        driver.manage().window().maximize();
```

Comment:
It's better to configure the driver in the `setUpDriver` method to keep all driver configurations in one place.